software/filephile.livecodescript

```diff
@@ -0,0 +1,150 @@
+﻿script "filephile"
+on devLog pPhase
+   --   -- ! commented out 15DEC2015 cuz logging slows things down a lot
```

Comment:
Will the lines added still be in the file if the app crashes before closing the file?